ppafm/common.py

```diff
@@ -67,7 +67,6 @@ class PpafmParameters(pydantic.BaseModel):
     colorscale_kpfm: str = "seismic"
     ddisp: float = 0.05
     aMorse: float = -1.6
-    tip_base: typing.List = ["None", 0.00]
```

Comment:
I had to do it, cause Toml was getting crazy trying to pass an inhomogeneous array.

At the same time, the parameter is not used anywhere (`get grep tip_base`) and was supposed to be [removed](https://github.com/Probe-Particle/ppafm/wiki/Setting-simulation-parameters#temporary-or-past-parameters).